packages/firebase_dynamic_links/lib/src/firebase_dynamic_links.dart

```diff
@@ -4,6 +4,9 @@
 
 part of firebase_dynamic_links;
 
+typedef Future<dynamic> OnLinkSuccessHandler(PendingDynamicLinkData linkData);
```

Comment:
I would use the newer `Function` declaration. It's now preferred by the dart team. [Style guide](https://dart.dev/guides/language/effective-dart/design#dont-use-the-legacy-typedef-syntax)

```dart
typedef OnLinkSuccessCallback = Future<dynamic> Fucntion(PendingDynamicLinkData linkData);
```

Also, I believe `Callback` is preferred over `Handler` for these. (e.g. [VoidCallback](https://api.flutter.dev/flutter/dart-ui/VoidCallback.html))

Reply:
Done!